docs/index.md

```diff
@@ -251,6 +248,131 @@ $ git clone https://github.com/grpc/grpc.git
   </div>
 </div>
 
+#### Install gRPC
+
+To run the example (and our tutorial examples, and any gRPC projects you create yourself), you need to install the *gRPC runtime* for your chosen language. In addition, if you want to try generating gRPC code, depending on your chosen language you may need to install the protocol buffers compiler with the appropriate gRPC plugin. We intend to further simplify this process in future releases.
+
+To install the gRPC runtime for your chosen language:
+<div class="tabs">
+  <ul>
+    <li><a href="#java_install">Java</a></li>
+    <li><a href="#cpp_install">C++</a></li>
+    <li><a href="#python_install">Python</a></li>
+    <li><a href="#go_install">Go</a></li>
+    <li><a href="#ruby_install">Ruby</a></li>
+    <li><a href="#node_install">Node.js</a></li>
+    <li><a href="#csharp_install">C#</a></li>
+    <li><a href="#objective-c_install">Objective-C</a></li>
+    <li><a href="#php_install">PHP</a></li>
+  </ul>
+<div id="java_install">
+This example is also part of Java gRPC itself's build, so running it is a little different to creating and running a regular project. To install and build it follow the instructions in the [Quickstart](https://github.com/grpc/grpc-java/tree/master/examples).
+
+To install the runtime for your own projects, add the following to your build files:
+
+Gradle:
+<pre>compile 'io.grpc:grpc-all:0.13.1'</pre>
```

Comment:
So grpc-java will need to bump this documentation with each release. I'm a bit concerned that it's going to bitrot, but we can try it out for a while. It does seem Java is the only language that will have a problem; I can't find a good link to mimic what C# is doing (all the links I have that are useful include the version number).